lib/arjdbc/jdbc/adapter.rb

```diff
@@ -289,12 +289,14 @@ def active?
 
       # @override
       def reconnect!
+        super
```

Comment:
ah ok, that makes sense - should review if this isn't relevant in current stable as well, thanks